src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -523,6 +525,45 @@ RequestContext<object> requestContext
             await requestContext.SendResult(result.First().ToString());
         }
 
+        private async Task HandleGetCommandRequest(
+            string param,
+            RequestContext<object> requestContext)
+        {
+            PSCommand psCommand = new PSCommand();
+            if (!string.IsNullOrEmpty(param)) {
+                psCommand.AddCommand("Microsoft.PowerShell.Core\\Get-Command").AddArgument(param);
+            }
+            else
+            {
+                psCommand.AddCommand("Microsoft.PowerShell.Core\\Get-Command")
+                        .AddParameter("CommandType", new[]{"Function", "Cmdlet", "ExternalScript"})
+                    .AddCommand("Select-Object")
+                        .AddParameter("Property", new[]{"Name", "ModuleName"})
+                    .AddCommand("Sort-Object")
+                        .AddParameter("Property", "Name");
+                // psCommand.AddScript("Microsoft.PowerShell.Core\\Get-Command -CommandType Function, Cmdlet, ExternalScript | Select-Object Name,ModuleName | Sort-Object Name");
```

Comment:
Nit: delete commented out code

Reply:
sssshhhhhhh this is not the code you're looking for 👋